README.md

```diff
@@ -100,7 +114,10 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema is a database structure created to hold data.
+
+One person can have many wishes
+
```

Comment:
What schema (tables and columns) would we use to represent that relationship?  Remember to discuss primary and foreign keys when describing relationships.